data/index.json

```diff
@@ -177,11 +177,12 @@
     "name": "big-decent-clipped-38",
     "type": "table",
     "id": "big-decent-clipped-38",
+    "separator": ",",
     "description": "",
     "creator": "Alexander Lex",
     "path": "data_38.csv",
     "size": [
-      19,
+      102,
```

Comment:
I think that the server should take care of determining the size of the data set. Setting this manually is error prone and tedious.